paper/src/main/java/com/denizenscript/denizen/paper/properties/EntityBeeStingers.java

```diff
@@ -0,0 +1,45 @@
+package com.denizenscript.denizen.paper.properties;
+
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizen.objects.properties.entity.EntityProperty;
+import com.denizenscript.denizencore.objects.Mechanism;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+
+public class EntityBeeStingers extends EntityProperty<ElementTag> {
+
+    // <--[property]
+    // @object EntityTag
+    // @name body_stingers
+    // @input ElementTag(Number)
+    // @plugin Paper
+    // @description
+    // Controls the number of bee stingers stuck in an entity's body.
+    // Note: Bee stingers will only be visible for players or player-type npcs.
+    // -->
+
+    public static boolean describes(EntityTag entity) {
+        return entity.isLivingEntity();
+    }
+
+    @Override
+    public ElementTag getPropertyValue() {
+        int stingers = getLivingEntity().getBeeStingersInBody();
+        return stingers == 0 ? null : new ElementTag(stingers);
```

Comment:
should use the isdefault method, otherwise the tag will be null here, which wouldn't be great